documentation/guide-repository.asciidoc

```diff
@@ -117,6 +117,59 @@ NOTE: In Quarkus, native and named queries via the `@Query` annotation are curre
 
 For Spring applications, devon4j offers a proprietary solution that integrates seamlessly with QueryDSL and uses default methods instead of the fragment approach. A separate guide for this can be found link:spring/guide-devon4j-spring-repository.asciidoc[here].
 
+=== Custom methods without fragment approach
+
+The fragment approach is a bit laborious, as always three types (repository interface, fragment interface and fragment implementation) are needed to implement custom methods.
```

Comment:
...as three types (repository interface, fragment interface and fragment implementation) are **always** needed to implement custom methods.